src/MeshVolume.js

```diff
@@ -108,6 +118,7 @@ export default class MeshVolume {
       specular: new Color(defaultMaterialSettings.specularColor),
       opacity: alpha,
       transparent: alpha < ALPHA_THRESHOLD,
+      side: DoubleSide,
```

Comment:
Just curious if this change is relevant to clipping?

Reply:
this change is relevant because now that objects can be clipped, the insides of their surfaces can be seen.  Without that line, the inside surfaces would be invisible due to WebGL backface culling (an optimization to save from rendering faces that are pointing away from the viewer and wouldn't normally be seen).

Reply:
Ah!! 💡